### DISCUSSION TITLE
Nearest Point Warnings

### DISCUSSION THREAD

**Discussion by ryancasburn-KAI** [blaylockbk/Herbie#184] [Q&A]
Is it expected to receive the three warnings from metpy and another warning from pyproj for each call of `.herbie.nearest_points`?

```
UserWarning: More than one time coordinate present for variable "gribfile_projection".
  warnings.warn('More than one ' + axis + ' coordinate present for variable'
UserWarning: More than one time coordinate present for variable "vis".
  warnings.warn('More than one ' + axis + ' coordinate present for variable'
UserWarning: More than one time coordinate present for variable "gribfile_projection".
  warnings.warn('More than one ' + axis + ' coordinate present for variable'
FutureWarning: Calling float on a single element Series is deprecated and will raise a TypeError in the future. Use float(ser.iloc[0]) instead
  return self._transformer._transform_point(
```

**Comment by jlounds** [comment]
Hello @ryancasburn-KAI -- were you able to find an answer to this? I am wondering the exact same thing!

**Comment by blaylockbk** [comment]
The warnings are expected. I haven't done any digging lately, but it would be nice to figure out if Herbie can do something to not throw warnings here. There does appear to be an issue when getting nearest_points for ECMWF fields as reported in https://github.com/blaylockbk/Herbie/issues/273. Haven't had time to do any digging, but maybe someday.
